<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>641</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Teste!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labeImageInfo">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNameInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelSizeInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWidthInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelHeightInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelWidth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelHeight">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>66</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameFiltersOptions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelFilter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Filter</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonHqx">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>hq4x</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonOriginal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Original</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonNearest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Nearest</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXbrZ">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>195</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>xBRZ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonScale2x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale2x</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonEagle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>287</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Eagle</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonVector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Vectorize</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonCRT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>339</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CRT</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxScaleFactor">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>362</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxNearestScaleFactor">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>95</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="scale2xSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>263</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Number of passes"/>
     </property>
     <property name="toolTipDuration">
      <number>4</number>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="eagleSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>312</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>312</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N. passes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>95</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Factor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>263</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N. passes:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>362</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Factor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Factor:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="xBRZSpinBox">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>220</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSuperSaI2x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>135</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SuperSaI2x</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="xBRSpinBox">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>174</y>
       <width>42</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>4</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonXbr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>149</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>xBR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>174</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Factor:</string>
     </property>
    </widget>
    <zorder>labelFilter</zorder>
    <zorder>radioButtonHqx</zorder>
    <zorder>radioButtonOriginal</zorder>
    <zorder>radioButtonNearest</zorder>
    <zorder>radioButtonXbrZ</zorder>
    <zorder>radioButtonScale2x</zorder>
    <zorder>radioButtonEagle</zorder>
    <zorder>radioButtonVector</zorder>
    <zorder>radioButtonCRT</zorder>
    <zorder>spinBoxScaleFactor</zorder>
    <zorder>spinBoxNearestScaleFactor</zorder>
    <zorder>scale2xSpinBox</zorder>
    <zorder>eagleSpinBox</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>label</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>xBRZSpinBox</zorder>
    <zorder>radioButtonSuperSaI2x</zorder>
    <zorder>graphicsViewOriginal</zorder>
    <zorder>xBRSpinBox</zorder>
    <zorder>radioButtonXbr</zorder>
    <zorder>label_6</zorder>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewOriginal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewGraph">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSimilarityGraphView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Similarity Graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOriginalView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilteredView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtered</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>39</y>
      <width>601</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="exportGIFButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export 
Animated GIF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1223</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
